examples/01_plotting/plot_3d_map_projection.py

```diff
@@ -0,0 +1,38 @@
+"""
+Making a surface plot of a 3D statistical map
+=============================================
+
+Download a statistical map from Neurovault and a cortical mesh (fsaverage),
+project the 3D data on the mesh and display a surface plot.
+
+"""
+
+from nilearn import datasets
+from nilearn import plotting
+
+##############################################################################
+# Get a statistical map
+# ---------------------
+
+brainpedia = datasets.fetch_neurovault_ids(image_ids=(32015,))
```

Comment:
Can you use the map in other visualization examples rather:
https://2698-1235740-gh.circle-artifacts.com/0/home/ubuntu/nilearn/doc/_build/html/auto_examples/01_plotting/plot_demo_glass_brain.html#sphx-glr-auto-examples-01-plotting-plot-demo-glass-brain-py

That way it's easier to compare examples. Also, this map is easy to understand (it's a simple button press map)